src/eei.cpp

```diff
@@ -28,17 +28,43 @@
 using namespace std;
 using namespace wasm;
 
+namespace {
+
+string to_string(evm_address const& address)
+{
+  stringstream s;
+  s << hex;
+  for (unsigned i = 0; i < 20; i++)
+    s << address.bytes[i];
+  return s.str();
+}
+
+string to_string(evm_uint256be const& value)
+{
+  stringstream s;
+  s << hex;
+  for (unsigned i = 0; i < 32; i++)
+    s << value.bytes[i];
+  return s.str();
+}
+
+}
+
+#if HERA_DEBUGGING
```

Comment:
Isn't this conditional backwards? You want to print if HERA_DEBUGGING is ON right?